docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -251,7 +252,7 @@ function my_plugin_blacklist_blocks() {
 	wp_enqueue_script(
 		'my-plugin-blacklist-blocks',
 		plugins_url( 'my-plugin.js', __FILE__ ),
-		array( 'wp-blocks' )
+		array( 'wp-blocks', 'wp-dom-ready' )
```

Comment:
```suggestion
		array( 'wp-blocks', 'wp-edit-post )
```

Reply:
Also note that the line 249 needs to be changed to: 

```php
// Plugin Name: My plugin
```

otherwise, WordPress won't pick up the plugin.

Reply:
Note that the `wp-edit-post` dependency is necessary to make sure the plugin code is executed after `_wpLoadGutenbergEditor` [is loaded](https://github.com/WordPress/gutenberg/blob/8790bce7ae56191d00fe7384ea1832df10624961/lib/client-assets.php#L1335).